### PR TITLE
bump dev vcs git lfs

### PR DESCRIPTION
**Pull request by holgersson32644**
- dev-vcs/git-lfs: Bump to 3.3.0
- dev-vcs/git-lfs: Update the live ebuild

### CODE REVIEW

File: dev-vcs/git-lfs/git-lfs-9999.ebuild [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
 if [[ "${PV}" = 9999* ]]; then
 	EGIT_REPO_URI="https://${EGO_PN}"
 	inherit git-r3
 else
-	SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+	SRC_URI="https://${EGO_PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	# Add the manually vendored tarball.
+	# Compress the tarball with: xz -9kT0 --memlimit-decompress=256M
+	SRC_URI+=" https://files.holgersson.xyz/gentoo/distfiles/golang-pkg-deps/${P}-deps.tar.xz"
```
Comment (thesamesam) [inline dev-vcs/git-lfs/git-lfs-9999.ebuild:24]: What's the story here? Did the upstream vendoring thing stop working?

Comment (holgersson32644) [reply dev-vcs/git-lfs/git-lfs-9999.ebuild:24]: So to say, upstream decided they won’t do that anymore. See the second paragraph of their [release notes](https://github.com/git-lfs/git-lfs/releases/tag/v3.3.0) (and the corresponding [PR](https://github.com/git-lfs/git-lfs/pull/4903) and [bug](https://github.com/git-lfs/git-lfs/issues/4827)):

> In this release, we no longer include vendored versions of our dependencies in
> the repository or the tarballs. These were a source of noise and bloat, and
> users can easily download the required dependencies with Go itself. Users who
> need to continue to vendor the dependencies can use the make vendor target.

Comment (holgersson32644) [reply dev-vcs/git-lfs/git-lfs-9999.ebuild:24]: Note that git-lfs-3.4.0 was just released, so please do not merge 3.3.0.

Comment (holgersson32644) [reply dev-vcs/git-lfs/git-lfs-9999.ebuild:24]: nvm, I did not see that both ebuilds were already merged.

**Pull request closed without merging**